freqtrade/exchange/exchange.py

```diff
@@ -24,6 +24,12 @@
 from freqtrade.exchange.common import BAD_EXCHANGES, retrier, retrier_async
 from freqtrade.misc import deep_merge_dicts
 
+
+# Should probably use typing.Literal when we switch to python 3.8+
```

Comment:
The point  here is not the "once we switch to python3.8" - it's "once we drop support for python <3.8" - which honestly, i don't see happen as there's nothing in 3.8 i think is a "must have" for us.

Maybe we should change the comment to "when support for <python3.8 is dropped" ? 

it's the same with 3.7 - but there's technically no reason to drop support for python3.6 at the moment as there've been no breaking changes.

Reply:
I mean, Literal is markes as if it added only in 3.8: https://docs.python.org/3/library/typing.html?highlight=typing#typing.Literal

Reply:
I do understand that - and i don't disagree with leaving this comment there.

But we do support 3.8 - but we also support 3.7/3.6 - so we can't use this. 
That's not clear from the comment above ... it sounds more like "once we do the transition from 2.7 to 3.x ..."

Reply:
okay, removed the comment since it's unclear.

I meant here that Literal is only available in 3,8+, so till we support 3.6 and 3.7, we cannot use Literal here instead of current Any.